Button.jsx

```diff
@@ -0,0 +1,13 @@
+
+import React from 'react';
+
+
+const Button = ({ onClick, children }) => {
+  return (
+    <button onClick={onClick}>
+      {children}

```

Comment:
And on line 8 of your button.jsx, children prop has no value, because you didn't pass any prop called children in your app.jsx.
I'm guessing "click me" is meant to be the children prop